web/e2e/dashboard.spec.ts

```diff
@@ -1,8 +1,8 @@
-import { expect, test } from "@reearth/e2e/utils";
+// import { expect, test } from "@reearth/e2e/utils";
 
-test("dasboard can be logged in", async ({ page, reearth }) => {
-  await reearth.initUser();
-  await reearth.goto(`/dashboard/${reearth.teamId}`);
+// test("dasboard can be logged in", async ({ page, reearth }) => {
+//   await reearth.initUser();
+//   await reearth.goto(`/dashboard/${reearth.teamId}`);
 
-  await expect(page.getByText(`${reearth.userName}'s workspace`)).toBeVisible();
-});
+//   await expect(page.getByText(`${reearth.userName}'s workspace`)).toBeVisible();
+// });
```

Comment:
Why should these been commented out?

Reply:
it is no use file but include old login test. So save it temporary, I add no use comment. Please check.
https://github.com/reearth/reearth-visualizer/pull/1140/commits/3261d87b82731dd830bbecad32c6194daed624bf